### PR TITLE
Ung 2096/create service to create uncompleted pocs

### PR DESCRIPTION
**Pull request by lalottacontinua**
Scheduled service is still missing, as well as mutliple tests.
But to make the frontend work properly with the endpoints, it would be good, to make this intermediate update.

### CODE REVIEW

File: src/main/resources/db/migration/V1_0__test_migration.sql [hunk 1]
```diff
@@ -31,8 +31,9 @@ CREATE TABLE IF NOT EXISTS poc_manager.poc_table
     manager_name                text            NOT NULL,
     manager_email               text            NOT NULL,
     manager_mobile_phone        text            NOT NULL,
-    role_and_group_name         text            NOT NULL,
-    group_path                  text            NOT NULL,
+    role_name                   text            NOT NULL,
+    device_group_id       text,
```
Comment (wojciechUrbanski) [inline src/main/resources/db/migration/V1_0__test_migration.sql:35]: Formatting is a little bit different here

File: src/main/scala/com/ubirch/controllers/TenantAdminController.scala [hunk 2]
```diff
@@ -163,7 +161,7 @@ class TenantAdminController @Inject() (
     token.roles.find(_.name.startsWith(TENANT_GROUP_PREFIX)) match {
 
       case Some(tenantRole) =>
-        tenantTable.getTenantByGroupId(TenantGroupId(tenantRole.name)).map {
+        tenantTable.getTenantByDeviceGroupId(TenantGroupId(tenantRole.name)).map {
```
Comment (wojciechUrbanski) [inline src/main/scala/com/ubirch/controllers/TenantAdminController.scala:164]: Is DeviveGroupId same thing as `TenantGroupId`? I mean, method is called `ByDeviceGroupId` but we are passing `TenantGroupId`, is it fine?

Comment (lalottacontinua) [reply src/main/scala/com/ubirch/controllers/TenantAdminController.scala:164]: We now have two different TenantGroups. One for the device realm and one for the user realm. When we don't use the name but the UUID to qurery the group, we will have to differentiate. :)

File: src/main/resources/db/migration/V1_0__test_migration.sql (hunk 1 above)
Comment (lalottacontinua) [reply src/main/resources/db/migration/V1_0__test_migration.sql:35]: Thanks, I will fix that!

**Pull request merged**